suites/suites/suites.yaml

```diff
@@ -121,6 +121,17 @@ templates:
     <<: *manager_blueprint_commercial_override
     node_templates.manager.properties.cloudify.workflows.task_retries: 100
 
+  - &aws_ec2_manager_blueprint_override
+    <<: *manager_blueprint_override
+    <<: *aws_ec2_manager_blueprint_sg_override
+
+  - &aws_ec2_manager_blueprint_sg_override
```

Comment:
`&aws_ec2_manager_blueprint_sg_override` should be placed before `aws_ec2_manager_blueprint_override`.
Specifically, put it next to the other security group override templates